index.html

```diff
@@ -1275,6 +1549,101 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 the cost of implementation complexity.
           </dd>
+          <dt id="pf20">
+PF20: Based on existing official standards.
+          </dt>
+          <dd>
+JSON and JWT are proposed and mature IETF standards. While JSON-LD 1.0 is in
+REC state in W3C, JSON-LD 1.1 is still in WD state. LD-Proofs are not standardized at all.
+          </dd>
+          <dt id="pf21">
+PF21: Designed to be small in size.
+          </dt>
+          <dd>
+JSON was invented as a simple data format to be transmitted on the wire. A verifiable credential
+can be expressed by its attributes only, without the necessity to introduce additional meta-information
+such as @context. This makes the resulting JSON+JWT credential typically also smaller in size.
+          </dd>
+          <dt id="pf22">
+PF22: Offline support without further processing.
+          </dt>
+          <dd>
+A JWT can fully describe itself without the need to retrieve or verify any external documents.
+JSON-LD requires the context to be queryable and requires further documents to be accessible
+to check the prevalent document, e.g., LD-Proof. Additional caching needs to be implemented
+to support offline use cases.
+          </dd>
+          <dt id="pf23">
+PF23: Wide adoption in other existing standards.
+          </dt>
+          <dd>
+JWT founds its application in many other existing standards, e.g., OAuth2, OpenID Connect.
+This allows for backward compatibility with existing authentication and authorization
+frameworks without or with only minor modifications to these legacy systems.
+          </dd>
+          <dt id="pf24">
+PF24: No type ambiguity.
+          </dt>
+          <dd>
+It is best practice that JSON data structures typically do not expect changing types of
+their internal attributes. JSON-LD has implicit support for compact form serialization
+which transforms arrays with a single element only to switch its data type. Developers
+writing parsers have to implement special handling of these data types, which results in
+more code, is more error-prone and sometimes does not allow parsers based on code
+generation, which rely on static types.
+          </dd>
+          <dt id="pf25">
+PF25: Broad library support.
+          </dt>
+          <dd>
+JWT and JSON due to its maturity and standardization, have a lot of open-source library
+support. While JSON-LD 1.0 is a standard and has support for different programming
+languages, it is still behind JSON which is often part of the native platform
+toolchain, e.g., JavaScript. For LD-Proofs, on the other hand, only a few scattered
+libraries exist.
+          </dd>
+          <dt id="pf26">
+PF26: Easy to understand what is signed.
+          </dt>
+          <dd>
+JWT makes it visible what is signed in contrast to LD-Proofs, e.g., LD Signatures, that
+are detached from the actual payload and contain links to external documents which makes
+it not obvious for a developer to figure out what is part of the signature.
```

Comment:
I could nitpick this one ... LD proofs sign everything in the document... JWT `kid` can be a link to an external document (and if using embedded keys the dev had better ensure the public key matches what is expected)... and so on. But I've been informed that there will be an editorial note indicating that these sections are for the reader to look over and make a decision about and that they might not be in agreement with one another.

Reply:
Yes, this is why it makes sense to have separate sections for tradeoffs.